### ISSUE TITLE
Feature request polynomial LR

### ISSUE THREAD

**Issue by ternaus** [catalyst-team/catalyst#464]
Would be nice to have a way to adjust learning rate with respect to the schedule:

```
lr = lr_base * (1 - (iter / iter_max)^alpha)
```

**Comment by Scitator** [comment]
Have you tried https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ExponentialLR ?

**Comment by ternaus** [comment]
Not the same. I would prefer learning rate to change every step and not every epoch.

**Comment by Scitator** [comment]
For this you can use `mode="batch"` for SchedulerCallback... with every PyTorch Scheduler :)

**Comment by Scitator** [comment]
Looks like everything works correctly, closed due to inactivity.
